backend/src/main/java/com/daily/daily/dailrypage/service/DailryPageService.java

```diff
@@ -28,13 +33,21 @@ public DailryPageDTO create() {
         return DailryPageDTO.from(savedPage);
     }
 
-    public DailryPageDTO create2(Long dailryId) {
-
```

Comment:
음 페이지 생성 메서드에서 dailryPageUpdateDTO를 받는 것이 좋을까요?

저는 페이지 생성은 빈 페이지만 만든다고 생각을 했었는데, 
혹시 여기서 dailryPageUpdateDTO를 받은 특별한 이유가 있으신가요?

Reply:
```
DailryPage dailryPage = DailryPage.builder()
                .dailry(dailry)
                .pageNumber(newDiaryPageNumber)
                .elements(elements)
                .build();
```
이 부분에 elements를 안넣어주면 

```
return DailryPageDTO.from(savedPage);
```
여기서 밑에 같이 에러가 나더라고요. dailry랑 매핑하고 pageNumber를 넣어주려면 위에 코드가 필요하다고 생각해서 일단은 element를 받아오는걸로 작업했거든요.
Cannot invoke "java.util.Map.values()" because the return value of "com.daily.daily.dailrypage.domain.DailryPage.getElements()" is null

Reply:
아하 그 문제는 제 생각에 아래와 같이 @Builder.Default를 붙이면 해결될 것 같습니다.

```java
    @Type(JsonType.class)
    @Column(name = "elements", columnDefinition = "JSON")
    @Builder.Default
    private Map<String, Object> elements = new HashMap<>();
```

Reply:
오 해결됐네요 감사합니다!